.transifexrc.tpl

```diff
@@ -1,4 +0,0 @@
-[https://www.transifex.com]
-hostname = https://www.transifex.com
-username = cozy
-token =
```

Comment:
Removing this file does not please travis: https://travis-ci.org/cozy/cozy-drive/builds/227905003#L405